docs/guide-bedtools.md

```diff
@@ -0,0 +1,101 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.3
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Emulating bedtools commands
```

Comment:
I'd suggest a section title more like:

"Bioframe for bedtools users"

Reply:
Done :)